### ISSUE TITLE
[Run] Show prompt output in run command for test and compile examples

### ISSUE THREAD

**Issue by jan25** [rust-lang/rustlings#1218]
`rustlings run test|compile_example` should output prompt to remove `I AM NOT DONE` comment in head of example file, to help users how to proceed to next example.

Unsure the historical reason behind why the `run` command for compile_example just prints success message and for test_example exits silently. I think we can make its output similar to `watch` command.

I can reopen earlier PR #176 if this suggestions make sense.

**Comment by diannasoreil** [comment]
I agree, but this should probably only apply when ran with `rustlings run next`, as the original purpose of `rustlings run` is to test solutions out-of-band for a specific exercise. `rustlings run next` is a fairly recent addition. Would be happy to review a PR for this!

**Comment by mo8it** [comment]
Should be resolved by the new handling in v6
